lib/patch_definition_table.erb

```diff
@@ -0,0 +1,51 @@
+<table style="white-space: normal">
+  <thead>
+  <tr>
+    <th>Path</th>
+    <th>Operation</th>
+    <th>Cardinality</th>
+    <th>Type</th>
```

Comment:
Update to
```
    <th>Value Cardinality</th>
    <th>Value Type</th>
```

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-bac15500a3095fd0a82601de2a06101eR6